processing/src/main/java/io/druid/query/filter/JavaScriptDimFilter.java

```diff
@@ -231,6 +231,12 @@ public boolean applyFloat(float input)
       };
     }
 
+    @Override
+    public DruidDoublePredicate makeDoublePredicate()
```

Comment:
Please make all methods in this class lambdas for consistency, and add "// Can't avoid boxing here" comment as above